examples/integrations/server.js

```diff
@@ -0,0 +1,141 @@
+const express = require("express");
+const { makeExecutableSchema } = require("graphql-tools");
+const queries = require("./queries");
+const axios = require("axios");
+const {
+  ApolloVoyagerServer,
+  gql
+} = require("../../packages/apollo-voyager-server");
+const { GraphQLScalarType } = require("graphql");
+const { Kind } = require("graphql/language");
+
+// Types
+const typeDefs = gql`
+  scalar IntOrUnknown
+
+  type CharacterInfo {
+    name: String
+    surname: String
+    version: Int
+    height: IntOrUnknown
+    mass: IntOrUnknown
+    hair_color: String
+    skin_color: String
+    eye_color: String
+    homeworld: String
+    gender: String
+    genderFromName: GenderInfo
+  }
+
+  type GenderInfo {
+    firstName: String
+    lastName: String
+    scale: Float
+    gender: String
+  }
+
+  type Query {
+    getCharacterInfo(id: ID!): CharacterInfo
+    getGenderFromName(name: String, surname: String): GenderInfo
+  }
+`;
+
+const getGenderFromName = async (obj, args, context, info) => {
+  try {
+    const { name, surname } = args;
+    const response = await axios.get(
```

Comment:
With axios entire example becomes very trivial, however example seems to be overly big. 
I'm just looking for opinion if we really need demo for this as opposed to just documentation